### ISSUE TITLE
CommandLine.swift:744:17: warning: 'init(contentsOf:)' is deprecated

### ISSUE THREAD

**Issue by weissi** [nicklockwood/SwiftFormat#628]
When running on `master` (6d651123c8eb72f530c088e2287b5ed9193d727e) with Swift 5.3

```
docker pull -q swiftlang/swift:nightly-5.3-bionic
docker run -it --rm --privileged -w "$PWD" -v "$PWD:$PWD" swiftlang/swift:nightly-5.3-bionic swift build
```

you'll get warnings:

```
$ jw-docker-swift-master swift build
`docker.io/swiftlang/swift:nightly-master-bionic
/Users/johannes/extsrc/SwiftFormat/Sources/CommandLine.swift:744:17: warning: 'init(contentsOf:)' is deprecated
        cache = NSDictionary(contentsOf: cacheURL) as? [String: String] ?? [:]
                ^
/Users/johannes/extsrc/SwiftFormat/Sources/CommandLine.swift:744:17: warning: 'init(contentsOf:)' is deprecated
        cache = NSDictionary(contentsOf: cacheURL) as? [String: String] ?? [:]
                ^
/Users/johannes/extsrc/SwiftFormat/Sources/CommandLine.swift:744:17: warning: 'init(contentsOf:)' is deprecated
        cache = NSDictionary(contentsOf: cacheURL) as? [String: String] ?? [:]
                ^
[19/19] Linking swiftformat
```

**Comment by nicklockwood** [comment]
Fixed in 0.44.11

**Comment by weissi** [comment]
thank you so much!
